### ISSUE TITLE
Cinnamon 3.2.7: some applets show old-style notifications

### ISSUE THREAD

**Issue by ACinnamonUserInUK** [linuxmint/Cinnamon#6538]
Screenshot:

![image](https://cloud.githubusercontent.com/assets/9998846/26132971/4e914ab0-3a93-11e7-8b56-9b53e8bca5ea.png)

The notifications for other applets work as they should:

![image](https://cloud.githubusercontent.com/assets/9998846/26132997/8d803d8a-3a93-11e7-923d-ca10e4910451.png)
EDIT: Actually, that turns out to be the _only_ icon that shows the right sort of notification. Perhaps I have the wrong version of some package installed (which, possibly, may in turn indicate a missing dependency or something of that sort). FURTHER EDIT (sorry): yet, in places other than the panel - in Nemo, say, and in my dock, Plank - I see only new style notifications.

I tried removing one of the affected applets (namely, Expo) from the panel and adding it again. So doing made no difference. Moreover, I've got the latest version of Expo (in that I used the update applets function and no updates showed).

It's _possible_ I caused this problem by installing and then removing Cinnamon 3.4.

Mint 18.1 x64,

**Comment by Odyseus** [comment]
Hello, @ACinnamonUserInUK.

First, some clarifications. Those aren't notifications, those are tooltips. The shield icon isn't an applet, it's the tray icon of the Update Manager, which is a GTK application.

The tooltips are styled by your theme, not by any applet nor application. GTK applications uses the style defined by the **gtk-2.0** and **gtk-3.0** folders inside your themes. Although, a lot of applications have customized tooltips. And applets uses the style for tooltips defined by the Cinnamon theme (**cinnamon** folder inside your theme).

Basically, if you want all of your tooltips to look the same, use the same them for both **Controls** (GTK applications) and **Desktop** (Cinnamon theme). Although, I must say, the tooltips of the Mint-X themes look different.

**Comment by JosephMcc** [comment]
This isn't a bug or issue in Cinnamon.

**Comment by ACinnamonUserInUK** [comment]
@Odyseus, @JosephMcc: thank you both for the clarifications.

I've set one and the same theme for Controls and for Desktop and now seemingly all of my notifications are tooltips look consistent (except that the tooltips are smaller than the notifications).

It's a shame I had to change my Desktop theme, though, because there were things that I liked about the old one . . And indeed the Mint-Y Desktop theme - to which I've switched - has a bug (which I reported some time ago) whereby the Mint Menu extends off the bottom of the screen.

**Comment by JosephMcc** [comment]
Sorry I don't remember the bug report about the menu going off the screen. That isn't a theme issue though. The menu has no real limitations in place for it's size. If you add enough categories or favorites it will get bigger than the screen. How many you can fit just depends on your screen res. There is no way around that without adding scrollbars in funky places.

**Comment by ACinnamonUserInUK** [comment]
Thanks for that Joseph. The original bug report - to which I've added something just now - is #6110.
